target/scripts/helpers/database/manage/postfix-accounts.sh

```diff
@@ -74,6 +74,10 @@ function _arg_expect_mail_account
 function _account_should_not_exist_yet
 {
   __account_already_exists && _exit_with_error "'${MAIL_ACCOUNT}' already exists"
+  if grep -q "^${MAIL_ACCOUNT}" "${DATABASE_VIRTUAL}"
```

Comment:
I remember, that we had a similar issue in the past:

`postfix-virtual.cf` looks like:

```
myalias@example.com user@example.com
```

The grep statement above would also match on:

myalias@example.company.com

Adding a space for example should fix it:

`if grep -q "^${MAIL_ACCOUNT} " "${DATABASE_VIRTUAL}"`

Reply:
That should be handled in both lookups then? But for accounts table, instead of space a `|`?

Reply:
You are right.